### DISCUSSION TITLE
cdc断流场景，hive的分区如何提交呢

### DISCUSSION THREAD

**Discussion by wangxiangyu** [apache/flink-cdc#1167] [Q&A]
大家有用mysql cdc同步到hive的场景吗？并且基于eventtime做分区提交。有一个问题，对于断流的情况，watermark不再推进，大家怎么做分区提交呢？

**Comment by shadon178** [comment]
hive应该不适合实时的场景吧，更多的应该使用Hudi、Iceberg等吧。
